pkg/operator/ceph/object/objectstore_test.go

```diff
@@ -32,6 +32,49 @@ import (
 	"k8s.io/client-go/kubernetes/fake"
 )
 
+const (
+	dashboardAdminCreateJSON = `{
```

Comment:
Do you need the entire blob? Just use the ones that you explicitly need, that'd make the test more readable.